rocketmq-console/src/main/java/org/apache/rocketmq/console/util/MsgTraceDecodeUtil.java

```diff
@@ -92,6 +100,11 @@
                     // add the context type
                     subAfterContext.setContextCode(Integer.parseInt(line[6]));
                 }
+                // compatible with the old version
+                if (line.length >= 9) {
```

Comment:
Could we use the constant while not a number(such as 9 or 15)to be forward compatible?